README.md

```diff
@@ -15,25 +15,25 @@ Travis CI (Linux) | [![Build Status](https://travis-ci.org/JaCzekanski/Avocado.s
 AppVeyor (Windows) | [![Build status](https://ci.appveyor.com/api/projects/status/h1cs3bj1vhskjxgx/branch/develop?svg=true)](https://ci.appveyor.com/project/JaCzekanski/avocado/branch/develop) | [Windows - develop](https://ci.appveyor.com/api/projects/JaCzekanski/avocado/artifacts/avocado.zip?branch=develop&job=Environment%3A+TOOLSET%3Dvs2017)
 
 
-Emulator is in very early stage of development. Despite that it is able to run few 3D games. [Game compability list](https://github.com/JaCzekanski/Avocado/wiki/Compability)
+Despite this emulator being in early development, some 3D games can run. [Game compability list](https://github.com/JaCzekanski/Avocado/wiki/Compability)
 
 
-Right now there are no SPU (so **no sound** except simple Audio CD in games), MDEC (**black screen instead of movies**), timer implementation is bad (**games fail to boot** or run at wrong speed).
+There is currently no SPU (**no sound** except simple in-game Audio CDs) or MDEC (**black screen instead of movies**). The timer implementation does not function properly (**games fail to boot** or run at wrong speed).
 
 ## Running
 
-Avocado require BIOS from real console placed in data/bios directory.
-On first run you'll be asked to select BIOS rom. This can be changed later using Options->BIOS or by modifying **config.json** file.
+Avocado requires the BIOS from real console in the data/bios directory.
+Selection of a BIOS rom will be required on the first run. The rom can be changed under Options->BIOS or by modifying the **config.json** file.
 
-Avocado doesn't support fast booting. [UniROM](http://www.psxdev.net/forum/viewtopic.php?t=722) can be used to work around this problem. Place .rom file in data/bios directory and modify **config.json**:
+Avocado doesn't support fast booting. [UniROM](http://www.psxdev.net/forum/viewtopic.php?t=722) can be used as a work around. Place the .rom file in the data/bios directory and modify **config.json**:
 ```
 "extension": "data/bios/unirom_caetlaNTSC_plugin.rom"
 ```
 
-To fastbook game press **Start** button (Enter by default) or **R2** (keypad *) to slowboot.
-You can run included Caetla with **Select** button (Right shift) and run .exe from disc directly. 
+Press the **Start** button (Enter by default) to fastboot, or **R2** (keypad *) to slowboot a game.
+You can run the included game *Caetla* with the **Select** button (Right shift) then run the .exe directly from the disk.
```

Comment:
Thanks for your contribution.

> You can run the included game *Caetla*

Caetla is an application, not a game. More info on this[site](https://gamehacking.org/wiki/Caetla).
![](https://raw.githubusercontent.com/wiki/simias/rustation/images/caetla-0.34.png)